### PR TITLE
chore: Remove old intake queue

### LINKED ISSUE #9746: Move event hashing to new framework
- Hasher object already exists
- To resolve this issue, remove hashing from the gossip thread, and wire new gossip events into the hasher, instead of directly into the `InternalEventValidator`

### LINKED ISSUE #10793: Remove old intake queue
- With the removal of legacy intake, its queue is no longer needed
- The unhandled task capacity of the hasher should be increased to 10k to match the size of the current intake queue

### PR DESCRIPTION
**Pull request by alittley**
- closes #10793 
- closes #9746

### CODE REVIEW

**Review by cody-littley — APPROVED** [review]

**Review by edward-swirldslabs — DISMISSED** [review]

File: platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformWiring.java [hunk 1]
```diff
@@ -439,6 +427,15 @@ public InputWire<Long> getPcesWriterRegisterDiscontinuityInput() {
         return pcesWriterWiring.discontinuityInputWire();
     }
 
+    /**
+     * Get a supplier for the number of unprocessed tasks in the hasher.
+     *
+     * @return a supplier for the number of unprocessed tasks in the hasher
+     */
+    public LongSupplier getHasherUnprocessedTaskCountSupplier() {
```
Comment (cody-littley) [inline platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformWiring.java:435]:
```suggestion
     */
    @NonNull
    public LongSupplier getHasherUnprocessedTaskCountSupplier() {
```

**Review by cody-littley — DISMISSED** [review]
LGTM aside from nitpick.

Does this alter the wiring diagram? If so, could you please post the updated picture? If not, no need to post the picture.

**Review by edward-swirldslabs — DISMISSED** [review]

**Comment by alittley** [comment]
> LGTM aside from nitpick.
> 
> Does this alter the wiring diagram? If so, could you please post the updated picture? If not, no need to post the picture.

There aren't any changes to the diagram

File: platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformWiring.java (hunk 1 above)
Comment (alittley) [reply platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformWiring.java]: Done

**Review by cody-littley — APPROVED** [review]

**Review by jeromy-cannon — DISMISSED** [review]

**Review by lpetrovic05 — APPROVED** [review]

**Review by jeromy-cannon — APPROVED** [review]

**Pull request merged**